query.go

```diff
@@ -11,9 +11,38 @@ import "C"
 
 import "unsafe"
 
+const (
+	// notmuch_query_search_messages and notmuch_query_search_threads
```

Comment:
Use the names for the go wrappers, rather than the c identifiers -- so in this case Query.SearchMessages.